components/Accordion/Accordion.tsx

```diff
@@ -1,57 +1,64 @@
-import React, { FC } from "react";
+import React, { FC } from 'react';
 import {
   Accordion as MuiAccordion,
   AccordionSummary,
   AccordionDetails,
   Typography,
   styled,
-} from "@mui/material";
-import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+} from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 import { useTheme } from "@mui/material/styles";
 
 // Define the props types
 interface AccordionProps {
   title: string;
   contents: string;
-  alignment?: "left" | "right";
+  alignment?: 'left' | 'right';
 }
 
 const CustomAccordion = styled(MuiAccordion)(({ theme }) => ({
-  "&:before": {
-    display: "none",
+  '&:before': {
+    display: 'none',
   },
   background: theme.palette.background.default,
+
   boxShadow: "none",
+
   borderBottom:
+
     theme.palette.mode === "light"
+
       ? "1px solid rgba(0, 0, 0, 0.12)"
+
       : "1px solid rgba(255,255,255, 0.12)",
+
   "& .MuiAccordionSummary-root": {
+
     backgroundColor: "rgba(0, 0, 0, 0)",
+
```

Comment:
Why were these lines added? Can you run the linter again? I noticed many unintentional linting changes in other files too